src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
```

Comment:
`3` is a magic number, you need to create a dedicated constant for the number of balls - you can declare it as:
```
private static final int NUMBER_OF_BALLS = 3;
```

Reply:
not fixed